tests/models/test_symbol_search.py

```diff
@@ -0,0 +1,22 @@
+from tastyworks.models.symbol_search import symbol_search
```

Comment:
Should we not move this file to a module outside of `models`? Reason being that it's a test helper file rather than something that does work for the `models` classes in this module. Is this code even usable by the user or is it code that you just used during development?

Reply:
I'd be okay moving it out of models, but I'm unsure where.  It's definitely usable, though not sure how much use people will get out of it.

Package structure is not my forte.  I'm not sure any classes actually use this anymore as opposed to when it was under the `Dough API`.  I don't have any historical data from the old endpoint to compare with, but as it is now, it's just a search feature that returns similar symbols compared to the query - eg. searching 'SPY' will return all symbols that contain those characters.

Reply:
I see.

Is this code even referenced anywhere else? If not, I think it makes sense to just delete it :)

Reply:
I think it's safe to say this endpoint isn't referenced.  I propose actually moving this under tastyworks_api since it's still a valid API endpoint and could be useful for package consumers.  I too don't find it very useful, but it works haha

Reply:
Yea, kill it with 🔥 :D

Reply:
🔥 Removed 🔥  If it ever becomes useful, it's easy enough to re-implement 😆